frontend/src/static/js/v_authorship.js

```diff
@@ -188,6 +191,46 @@ window.vAuthorship = {
       }
     },
 
+    updateFilterSearch(evt) {
+      this.filterSearch = (evt.target.value.length !== 0) ? evt.target.value : '*';
+    },
+
+    getFilteredFiles() {
+    },
```

Comment:
this is here because?

Reply:
I tried to do ```v-on:submit.prevent``` for glob filter without specifying any method but it wasn't working so i had to add this method in.

Reply:
sflr; pls add a comment here if needed. but try this first, it should work, using vanilla form functions:

```
form(onsubmit="return false")
```